src/gui/helpers/debug_helper.rs

```diff
@@ -63,7 +63,7 @@ impl MovementGui {
         self.last_boat = bmd.position;
     }
 
-    fn draw_coord(&self, ui: &mut egui::Ui, pos: &Vector3<f32>) {
+    pub fn draw_coord(ui: &mut egui::Ui, pos: &Vector3<f32>) {
```

Comment:
borrowed this method for now, assuming you'll want to refactor this to suit your needs.